Documentation/network/servicemesh/ingress.rst

```diff
@@ -19,6 +19,24 @@ routing and for TLS termination.
     The ingress controller creates a service of LoadBalancer type, so
     your environment will need to support this.
 
+Cilium allows you to specify load balancer mode for the Ingress resource:
+
+- ``dedicated``: The Ingress controller will create a dedicated loadbalancer
+  for the ingress.
+- ``shared``: The Ingress controller will use a shared loadbalancer for all
+  Ingress resources.
+
+Each load balancer mode has its own benefits and drawbacks. The shared mode saves
+resources by sharing a single LoadBalancer config across all Ingress resources in
+the cluster, while the dedicated mode can help to avoid blast radius of a single
+Ingress resource affecting others.
+
+.. Warning::
+
+    Changing the load balancer mode for an Ingress resource is supported, but
+    there is a risk of losing connectivity to backend services due to new
+    load balancer IP addresses being assigned.
```

Comment:
In general I would prefer to avoid wording like "is supported" in docs like this, as it's not describing the intended behaviour of the system but it paints a vaguer picture about the relationship between the users and the code.

```suggestion
    It is possible to change the load balancer mode for an Ingress resource.
    When the mode is changed, active connections to backends of the Ingress
    may be terminated during the reconfiguration due to a new load balancer
    IP address being assigned to the Ingress resource.
```

Reply:
^ Put this way, I wonder if maybe an info callout is more appropriate rather than a warning. I don't think that this documentation is assuming that the user will be changing the load balancer mode at runtime and it's not explicitly telling the user to change this setting.

Warning would be used when there's something pretty unexpected and concerning to call out. I don't think users will be surprised that changing the mode of the Ingress will cause disruption to the traffic to that Ingress.

(I also considered whether it even needs to be a callout or if it could just be regular text in the body of this guide. This seems a bit more like exposition rather than being part of the narrative of the guide, a `.. note::` seems more appropriate.)

Reply:
oh, and I used `for an Ingress resource` above but I think it's a cilium-agent CLI argument? So maybe `for Ingress resources` would be more accurate if it cannot be configured for each Ingress.

Reply:
thanks a lot :100: